### PR TITLE
Added wishlist page

### PR DESCRIPTION
**Pull request by psychoactiv**
- Added wishlist page
- Added wishlist functionality
- Added quatity limit to cart page
- Added child Wish icon toggle functionality
- If added to wishlist and then added to cart then it'll be removed from cart and vice versa.
28 03 2022_20 47 57_REC
28 03 2022_20 47 39_REC

### CODE REVIEW

**Review by ShreyashMahajan — APPROVED** [review]
Nice to see that you have broken your code into small compoents.
And all the data is being called very systematically.
Good to merge

File: src/components/cartCard/cart-card.jsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+                          }
+                        >
+                          +
+                        </button>
+                      ) : (
+                        <button
+                          className="increase-cart-item-qty display-btn snck change-btn qty-btn"
+                          onClick={() => {
+                            decideSnackbarFate("INCREASE");
+                            console.log(decideSnackbarFate);
```
Comment (atharvabhanage02) [inline src/components/cartCard/cart-card.jsx:55]: You can remove the console.log before merging the PR

**Review by atharvabhanage02 — APPROVED** [review]
Just one very minor suggestion below ,  apart from that everything looks fine 

- Code is clean and readable
- Liked the way you used Loader for showing Loading State
- Cart functionality is working fine and the UI looks clean

File: src/components/cartCard/cart-card.jsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+                      )}
+                      {item.qty > 1 ? (
+                        <button
+                          className="decrease-cart-item-qty qty-btn"
+                          onClick={() => {
+                            cartDispatch({
+                              type: "DECREMENT_CART_ITEMS",
+                              payload: item.id,
+                            });
+                            console.log(decideSnackbarFate);
```
Comment (shaksham08) [inline src/components/cartCard/cart-card.jsx:69]: Remove console log

**Review by shaksham08 — APPROVED** [review]
Nicely written but remove the console log if not required

**Review by Omkar-Chaskar — APPROVED** [review]
Clean code and nicely structured components .
Easy to understand Logic. 
Nicely Done and Good to merge

File: src/components/cartCard/cart-card.jsx (hunk 1 above)
Comment (psychoactiv) [reply src/components/cartCard/cart-card.jsx:55]: sure, will do that

File: src/components/cartCard/cart-card.jsx (hunk 2 above)
Comment (psychoactiv) [reply src/components/cartCard/cart-card.jsx:69]: sure, on it

**Pull request merged**